lerobot/scripts/visualize_image_transforms.py

```diff
@@ -118,6 +119,25 @@ def save_config_single_transforms(cfg, original_frame, output_dir):
             transformed_frame = tf(original_frame)
             to_pil(transformed_frame).save(output_dir_single / f"{i}.png", quality=100)
 
+        # Apply min transformation
+        min_value, max_value = cfg[f"{transform}"].min_max
+        kwargs = {
+            f"{transform}_weight": cfg[f"{transform}"].weight,
+            f"{transform}_min_max": (min_value, min_value),
+        }
+        tf = get_image_transforms(**kwargs)
+        transformed_frame = tf(original_frame)
+        to_pil(transformed_frame).save(output_dir_single / "min.png", quality=100)
+
+        # Apply max transformation
+        kwargs = {
+            f"{transform}_weight": cfg[f"{transform}"].weight,
+            f"{transform}_min_max": (max_value, max_value),
+        }
+        tf = get_image_transforms(**kwargs)
+        transformed_frame = tf(original_frame)
+        to_pil(transformed_frame).save(output_dir_single / "max.png", quality=100)
+
```

Comment:
Could you add this one as well?
```suggestion
        # Apply mean transformation
        mean_value = (min_value + max_value) / 2
        kwargs = {
            f"{transform}_weight": cfg[f"{transform}"].weight,
            f"{transform}_min_max": (mean_value, mean_value),
        }
        tf = get_image_transforms(**kwargs)
        transformed_frame = tf(original_frame)
        to_pil(transformed_frame).save(output_dir_single / "mean.png", quality=100)
```

Reply:
f98c9a71e2cc679fb43cbe50fd6181ab22a10493 done